contentcuration/contentcuration/frontend/settings/pages/SettingsIndex.vue

```diff
@@ -65,10 +65,10 @@
       },
     },
     created() {
-      this.fetchDeferredUserData();
+      this.fetchDeferredUserApiToken(), this.fetchDeferredUserStorageByKind();
```

Comment:
While use of the comma `,` here does what you're intending, it is unnecessary. Other developers might be confused as to reason why it was used. So please place them on different lines and use a semicolon.